Bottom: adf8b0d94c0713a83e84d1f93421d5511bfffb32
Top:    79eb0402aaf0d10b6835d2b6a2736d88ea515e91
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-15 01:50:06 +0100

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index 9b2f76e..296eeeb 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -184,7 +184,7 @@ DrawStrippedText (ElementTypePtr Element, int min_width)
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   if (doing_pinout || doing_assy)
